composer.json

```diff
@@ -3,37 +3,35 @@
     "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
     "type": "project",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.1.3",
         "ext-curl": "*",
         "ext-fileinfo": "*",
         "ext-intl": "*",
         "ext-zip": "*",
         "beberlei/doctrineextensions": "^1.0",
         "composer/ca-bundle": "^1.0",
         "composer/installers": "^1.0.21",
-        "csa/guzzle-bundle": "~1.3",
+        "csa/guzzle-bundle": "~3.1",
         "cssjanus/cssjanus": "dev-patch-1",
         "curl/curl": "^1.2.1",
         "defuse/php-encryption": "~2.0.1",
-        "doctrine/cache": "^1.6",
-        "doctrine/common": "~2.5.3",
-        "doctrine/dbal": "^2.5.3",
+        "doctrine/cache": "^1.8",
         "doctrine/doctrine-bundle": "^1.6",
-        "doctrine/orm": "~2.5.3",
+        "doctrine/orm": "^2.5.11",
         "ezyang/htmlpurifier": "^4.10",
         "geoip2/geoip2": "~2.4.2",
         "greenlion/php-sql-parser": "^4.1",
-        "guzzlehttp/guzzle": "~5.0",
+        "guzzlehttp/guzzle": "~6.1",
```

Comment:
Can't we keep 5.x? It seems that many modules use Guzzle 5 to be compatible with previous PS versions

Reply:
about that, is it possible in 1.7.6 to use our own Guzzle version declared in composer, in module? i guess... not? i remember few discussions on StackOverflow, when developer needed to use different version of Guzzle in his own module but it was impossible due to conflicts

Reply:
> Can't we keep 5.x? It seems that many modules use Guzzle 5 to be compatible with previous PS versions

Sadly, I don't think it's possible: except if we remove the guzzle bundle and create our own services "hand by hand". See the requirements of the bundle on packagist => https://packagist.org/packages/csa/guzzle-bundle

@kpodemski I don't know, but I don't see a valid reason to use a specific version of guzzle when the core already provide one :/

Reply:
Upgrading Guzzle will be a major problem for modules who need to target all our current installed base, the will need to distribute separate zip files for PS <= 1.7.6

Is there a specific reason to update guzzle-bundle? Because otherwise we'll have to fork it. I think upgrading Guzzle in 1.7 is not a good idea.

Reply:
> Is there a specific reason to update guzzle-bundle

yes, the minimal php version supported :+1: 

For some who will complain we drop v5, others ones will complain we keep v5: https://stackoverflow.com/questions/53831504/prestashop-guzzle-conflict